runtime/battery-station/src/xcm_config/fees.rs

```diff
@@ -68,7 +67,7 @@ impl<
         let foreign_decimals = metadata.decimals;
 
         let fee_unadjusted = if let Some(fee_factor) = metadata.additional.xcm.fee_factor {
-            bmul(default_per_second, fee_factor).ok()?
+            default_per_second.bmul(fee_factor).ok()?
```

Comment:
Can you explain how you imported the `bmul` operation?

Reply:
Ah I see, the battery station uses the common runtime and in this you have imported `use zeitgeist_primitives::fixed::FixedMul;` 👍 cool